README.md

```diff
@@ -19,7 +21,9 @@ After downloading the module in the modules folder, visit the Extend (/admin/mod
 
 ## Configuration
 
-@ToDo
+1. Enable the module
+2. visit /media/add to add media content to website.
+3. Visit /admin/structure/media to edit the default media bundles.
 
```

Comment:
No i didn't. We have included slick module in dependency so those instructions are useless now. I'll remove them from media library PR